### DISCUSSION TITLE
Failed to apply for student certification

### DISCUSSION THREAD

**Discussion by dunxuan** [community/community#27566] [GitHub Education]
Failed to apply for student certification.What can i do?

**Comment by FahdSeddik** [comment]
Hey!
You have to make sure you're using your student email that has these benefits. You can find more details [here](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/why-wasnt-my-application-for-a-student-developer-pack-approved). Hope it helps!

**Reply by FahdSeddik** [reply]
Make sure to mark it as an answer so people would see it easier.

**Reply by dunxuan** [reply]
Our school email is temporarily unavailable. Is there any other way?

**Reply by FahdSeddik** [reply]
Contact administration of your school and contact GitHub support [here](https://support.github.com/contact/account?tags=hubberfy_account)

**Comment by mpboom** [comment]
Also note that on the forums, we sadly cannot address individual account issues or problems with the Student pack.
